front/components/actions/tables_query/TablesQueryActionDetails.tsx

```diff
@@ -166,8 +156,17 @@ function QueryTablesResults({ action }: { action: TablesQueryActionType }) {
   const title = output?.query_title ?? "query_results";
 
   return (
-    <div onClick={() => handleDownload(title)}>
-      <Citation size="xs" title={title} />
+    <div>
+      <Collapsible defaultOpen={true}>
```

Comment:
defaultOpen doesn't make much sense here, since this collapsible is already wrapped into another collapsible that itself honors the passed `defaultOpen`. If I decide to open the tables query thing, then I expect results to be default open (especially as it's a very small thing).